### PR TITLE
Snackify flash messages

### PR DESCRIPTION
**Pull request by HSalmon**
@gdpelican, @robguthrie, could probs do with some eyez :eyes:

### CODE REVIEW

File: config/locales/client.en.yml [hunk 1]
```diff
@@ -173,6 +174,8 @@ en:
     select_position: 'Select position'
     statement_placeholder: 'Share your thoughts with the group.'
     aria_statement_label: 'Optionally enter a statement to be recorded with your position'
+    messages:
+      created: Vote added
```
Comment (gdpelican) [inline config/locales/client.en.yml]: For consistencies sake, can we make a habit of putting quotes around yaml messages?

File: lineman/app/components/flash/flash.scss [hunk 2]
```diff
@@ -1,29 +1,21 @@
...
+  bottom: 20px;
+  left: 20px;
+  min-width: 220px;
+  position: fixed;
+  z-index: 200;
 }
 
 .flash-container {
-  padding: 5px 0;
+  background-color: $primary-text-color;
```
Comment (gdpelican) [inline lineman/app/components/flash/flash.scss]: Hmm.. why is the background a text color?

Actually, a screenshot of this would be baller.

**Comment by gdpelican** [comment]
A+++++++ Didn't make :eyes: melt WOULD REVIEW AGAIN A+

File: lineman/app/components/flash/flash.haml [hunk 3]
```diff
@@ -2,5 +2,3 @@
   .flash-container{ng-show: 'display()', ng-animate: "'enter'", class: 'alert-{{flash.level}}'}
     .flash-message
       {{ flash.message | translate:flash.options }}
-    %button.close.dismiss-flash.pull-right{type: 'button', ng-click: 'dismiss()'}
```
Comment (gdpelican) [inline lineman/app/components/flash/flash.haml]: Is there some code somewhere in here which makes the message actually go away? I guess I was expecting a javascript timeout or a css transition of some kind.

File: config/locales/client.en.yml (hunk 1 above)
Comment (HSalmon) [inline config/locales/client.en.yml]: Do you want me to go through and change that now?

Comment (robguthrie) [inline config/locales/client.en.yml]: Hrmm.. I'm not super keen on this always quote rule. I like brevity and yml has already thought about this.

**Comment by gdpelican** [comment]
I think it's a thing we need to decide on, because it's half on and half
off at the moment.

I prefer quotes because they always work, whereas no quotes breaks when you
need to interpolate and such.

**Pull request merged**